docs/faq.md

```diff
@@ -200,9 +203,49 @@ kubectl patch serviceaccount default \
 
 See more at [Using Google Container Registry (GCR) with Minikube](https://ryaneschinger.com/blog/using-google-container-registry-gcr-with-minikube/).
 
-### How do I enable debugging?
```

Comment:
We should probably still keep this entry as a placeholder to remind us to add something about enabling debugging (via the debugger agent and port forwarding). Can be in an HTML comment block, though.